source/elements/oneDAL/source/introduction/classes.rst

```diff
@@ -0,0 +1,4 @@
+Classes of ML algorithms
```

Comment:
You have two sections one right after the other, but one has Machine Learning spelled out, and the other has it abbreviated. I'd suggest to spell this one out as well.

Reply:
Fixed